src/main/java/com/oneops/secrets/proxy/SecretsUtils.java

```diff
@@ -178,13 +178,15 @@ private static void writeToken(String tokenName, String token) {
    * @param user user who we are prompting a password for
    * @return user-inputted password
    */
-  private static char[] readPassword(String user) {
+  private static char[] readPassword(String user, String pwd) {
     Console console = System.console();
     if (console != null) {
       System.out.format("Password for %s : ", bold(green(user)));
       return console.readPassword();
```

Comment:
if `pwd` is not empty (ie user intentionally gave password as part of the command), why should it ask for it again?

Reply:
It should not ask for password again. So before check the System.console, code reads for bearerToken, if it is null then only it will process readPassword else it will be see bearerToken and can return secretClients object.

Reply:
@sureshg : Updated condition as we discussed.